plugins/woocommerce-blocks/tests/e2e/tests/checkout/checkout-block.shopper.block_theme.side_effects.spec.ts

```diff
@@ -430,7 +430,89 @@ test.describe( 'Shopper → Checkout block → Place Order (guest user)', () =>
 	} );
 } );
 
-test.describe( 'Checkout Form Errors (guest user)', () => {
+test.describe( 'Shopper → Place Virtual Order', () => {
+	test.beforeEach( async ( { requestUtils } ) => {
```

Comment:
What do you think about making this a `beforeAll` instead of `beforeEach`? Both tests need the same value set, so it doesn't need to be reset between tests, and it would speed things up a little by eliminating a network request.